packages/driver/src/cy/commands/navigation.ts

```diff
@@ -325,21 +325,23 @@ const stabilityChanged = (Cypress, state, config, stable) => {
   }
 
   const loading = () => {
-    const href = state('window').location.href
-    const count = getRedirectionCount(href)
-    const limit = config('redirectionLimit')
+    if (state('window')) {
```

Comment:
Suggest adding a comment when window is not set.

Reply:
```suggestion
    // The window state will not exist if the the window that has loaded is cross origin
    if (state('window')) {
```